addons/web_editor/views/snippets.xml

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 <template id="snippets">
+    <t t-call="web_editor.snippets_content" t-lang="env.user.lang"/>
+</template>
+
+<template id="snippets_content">
```

Comment:
I don't see why you need to do that for the snippet panel rendering in fact. Those templates are rendered via a `render_template` call, can't you just use to user language there ?

Reply:
The problem is that the lang of the context is the wrong one (the default website lang) and the session.user_context doesn't have lang.

I could do a rpc using the user id and getting his lang but that would add a query. What do you think?